src/IdeaStatiCa.Plugin/BIMPluginHosting.cs

```diff
@@ -171,6 +225,7 @@ private void IS_Exited(object sender, EventArgs e)
 
 		protected void NotifyAppStatusChanged(AppStatus newStatus)
 		{
+			ideaLogger.LogDebug($"NotifyAppStatusChanged service '{ServiceBaseAddress}' newStatus = '{newStatus}'");
 			AppStatusChanged?.Invoke(this, new ISEventArgs() { Status = newStatus });
 		}
```

Comment:
It might be practical to track the end of the handling notification processing to see, where the handlers stopped their part.

```suggestion
			ideaLogger.LogTrace($"NotifyAppStatusChanged service '{ServiceBaseAddress}' newStatus = '{newStatus}' - handling of the event finished.");
		}
```